.github/workflows/js-lint.yml

```
@@ -0,0 +1,59 @@
+name: Lint JS
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches-ignore:
+      - translations
+
+jobs:
+  see_if_should_skip:
+    runs-on: ubuntu-latest
+
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@36feb0d8d062137530c2e00bd278d138fe191289
+        with:
+          cancel_others: 'false'
+          github_token: ${{ github.token }}
+          paths: '["**/*.js","package*.json",".github/workflows/js-lint.yml"]'
```

Let's add some spaces here for legibility. ✨ 

```suggestion
          paths: '["**/*.js", "package*.json", ".github/workflows/js-lint.yml"]'
```